pkg/occlient/occlient.go

```diff
@@ -550,9 +550,19 @@ func (c *Client) CreateNewProject(projectName string, wait bool) error {
 			if !ok {
 				break
 			}
-			if e, ok := val.Object.(*projectv1.Project); ok {
-				glog.V(4).Infof("Project %s now exists", e.Name)
-				return nil
+			if prj, ok := val.Object.(*projectv1.Project); ok {
+				glog.V(4).Infof("Status of creation of project %s is %s", prj.Name, prj.Status.Phase)
+				switch prj.Status.Phase {
+				//prj.Status.Phase can only be "Terminating" or "Active" or ""
+				case "Active":
```

Comment:
please use [`corev1.NamespaceActive`](https://github.com/openshift/odo/blob/3a2ca54ad9f93ec0659311c0cf4526aa783e5ef3/vendor/k8s.io/api/core/v1/types.go#L4177) instead of hardcored string